src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
+  grid-template-rows: auto;
+  min-width: 1900px;
+}
+
+.tableRow {
+  border-bottom: 1px solid $dark-grey;
```

Comment:
By looking at the XD, it looks like the border needs to be slightly transparent.